source-container-build/app/source_build.py

```diff
@@ -211,7 +224,17 @@ def skopeo_copy(
         flags.append("--remove-signatures")
     cmd = ["skopeo", "copy", *flags, src, dest]
     logger.debug("copy image: %r", cmd)
-    run(cmd, check=True)
+
+    try:
+        run(cmd, stderr=subprocess.PIPE, check=True)
+    except CalledProcessError as e:
+        is_missing_signatures = (
+            "Source image rejected: A signature was required, but no signature exists"
+            in e.stderr.decode()
+        )
+        if is_missing_signatures:
+            raise NoSignatureError
```

Comment:
```suggestion
            raise NoSignatureError(e.stderr.decode())
```
(tied to the previous suggestion)

Reply:
Thanks for the suggestion!